openstudio-ee.gemspec

```diff
@@ -28,8 +28,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.4.3'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.14'
+  spec.add_dependency 'openstudio-extension', '~> 0.4.4'
+  spec.add_dependency 'openstudio-standards', '~> 0.2.15.pre.rc1'
```

Comment:
Are we sure we want to release an actual version of this that points to a `pre.rc1` release of standards?

Reply:
@nllong Thanks for the catch, I'll update both gems here

Reply:
@nllong hold off something is wrong here. The change log and version object are also outdated, I thought I based this branch off of the testing branch. I'll make more more commit to fix things